### DISCUSSION TITLE
Polars dataframe native support

### DISCUSSION THREAD

**Discussion by M0nteCarl0** [catboost/catboost#2699] [Ideas]
Dear friends
Are you planning add support Polars dataframe support in Catboost?

**Comment by andrey-khropov** [comment]
Yes: https://github.com/catboost/catboost/issues/2524. But no concrete timeframe at this moment.

**Reply by M0nteCarl0** [reply]
I can help to implenting this feature  but only in the end of this year.

**Comment by jasonsross** [comment]
any update on this?

**Reply by andrey-khropov** [reply]
> any update on this?

I'm working on it at the moment.

**Comment by andrey-khropov** [comment]
Polars support have been added in the main branch in https://github.com/catboost/catboost/commit/9017641d98bf4fc687277926e35c542320405d2f. The next release is expected within 2-3 days.
